spec/features/merchants/invoices/show_spec.rb

```diff
@@ -40,4 +52,25 @@
       expect(page).to have_content(@customer1.last_name)
     end
   end
+
+  describe 'User Story 16' do
+    it "Displays an item name" do
+      visit merchant_invoice_path(@merchant1, @invoice1)
+
+      expect(page).to have_content(@item1.name)
+      expect(page).to_not have_content(@item5.name)
+    end
+
+    it "Displays a invoice items quantity, unit price, and status" do
+      visit merchant_invoice_path(@merchant1, @invoice1)
+
+      expect(page).to have_content(@invoice_item1.quantity)
+      expect(page).to have_content(@invoice_item1.unit_price)
+      expect(page).to have_content(@invoice_item1.status)
+
+      expect(page).to_not have_content(@invoice_item2.quantity)
```

Comment:
This test is failing sometimes.  I think if we wrap it in a within block we can fix that though.